src/Event/PurlNodeContextRoutes.php

```diff
@@ -57,10 +66,13 @@ public function __construct(EntityTypeManagerInterface $entity_type_manager, Rou
   public function purlCheckNodeContext(GetResponseEvent $event, $eventName, EventDispatcherInterface $dispatcher_interface) {
     $route_options = $this->routeMatch->getRouteObject()->getOptions();
     $isAdminRoute = array_key_exists('_admin_route', $route_options) && $route_options['_admin_route'];
+    $entity = $this->routeMatch->getParameter('node');
+    $entity = is_numeric($entity) ? $this->nodeStorage->load($entity) : $entity;
 
     if (!$isAdminRoute
       && $matched = $this->matchedModifiers->getMatched()
-      && $entity = $this->routeMatch->getParameter('node')
+      && $entity = $this->routeMatch->getParameter('node') && $entity = $this->routeMatch->getParameter('node')
```

Comment:
```
&& $entity = $this->routeMatch->getParameter('node') && $entity = $this->routeMatch->getParameter('node')
```

looks incorrect. did you forget removing these? I believe these are not necessary.

Reply:
Yes, that was incorrect. Thank you for finding that out.